benefits/templates/admin/agency-base.html

```diff
@@ -8,6 +8,10 @@
           rel="stylesheet"
           integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
           crossorigin="anonymous">
+    <script nonce="{{ request.csp_nonce }}"
+            src="https://cdn.jsdelivr.net/npm/jquery@3.6.1/dist/jquery.min.js"
+            integrity="sha256-o88AwQnZB+VDvE9tvIXrMQaPlFFSUTR+nldQm1LuPXQ="
+            crossorigin="anonymous"></script>
 {% endblock extrastyle %}
 
```

Comment:
@angela-tran Should I also add this line from `/calitp/app/benefits/core/templates/core/base.html` to pull in the recaptcha script, in this PR?

```liquid
    {% if request.recaptcha %}
      <script nonce="{{ request.csp_nonce }}" src="{{ request.recaptcha.script_api }}"></script>
    {% endif %}
```

Reply:
I would have preferred to be able to add this to the very bottom of the `<body>` tag, but there isn't an easy place to override and add this.

Reply:
FWIW, I think we can ignore reCAPTCHA for this effort -- the user must have access to the Admin, and I think we can assume they are legit if they can get in and initiate an in-person flow.